script.js

```diff
@@ -0,0 +1,40 @@
+window.onload = function () {
```

Comment:
Te sugiro utilizar o window.onload apenas para chamar as funções. Facilitaria bastante na hora de ler o código, e deixa ele mais limpo!
```suggestion
window.onload = function () {
  textInput();
  memeInput();
  // ...
}
```